tests/testapp/tests/test_uuid_utilities.py

```diff
@@ -1,29 +1,43 @@
+import hashlib
```

Comment:
Many of the changes here are just from running "black". I'll flag the new stuff.